contracts/mvx-game-sc/src/owner.rs

```diff
@@ -35,7 +35,7 @@ pub trait OwnerModule: crate::private::PrivateModule + crate::storage::StorageMo
 
                         for (winner, percentage) in val.into_iter() {
                             let reward_per_winner =
-                                &BigUint::from(percentage) * &total_wager / &BigUint::from(100u64);
+                                &BigUint::from(percentage) * &total_wager / &BigUint::from(10000u64);
```

Comment:
Rust accepts `_` character in integers, so best practice are:
*  `10_000u64`
* declare a constant with an intuitive name